thirdparty_containers/wycheproof/build.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<project name="wycheproof" default="build" basedir=".">
+	<taskdef resource="net/sf/antcontrib/antlib.xml" />
+	<description>
+		Build wycheproof Docker image
+	</description>
+
+	<!-- set properties for this build -->
+	<property name="DEST" value="${BUILD_ROOT}/thirdparty_containers/wycheproof" />
+	<property name="src" location="." />
+	<property name="jvm_version" location="${JVM_VERSION}" />
```

Comment:
Do we need property  `jvm_version`? I do not see it used anywhere.

Reply:
Thanks for the catch! I've removed `jvm_version`. 

This needs to be removed from the rest of the external tests as well -- as it's not used in any one of them, even though we've been copy-pasting it over in all.